example_configs/kafka-connect.yml

```diff
@@ -0,0 +1,100 @@
+lowercaseOutputName: true
+rules:
+  #kafka.connect:type=app-info,client-id="{clientid}"
+  #kafka.consumer:type=app-info,client-id="{clientid}"
+  #kafka.producer:type=app-info,client-id="{clientid}"
+  - pattern: 'kafka.(.+)<type=app-info, client-id=(.+)><>([a-z-]+): (.+)'
+    name: kafka_$1_info
```

Comment:
If this is kafka connect specific (as against general kafka library metrics), then is should be kafka_connect_ to distinguish them from general producer/consumer/kafka server metrics.

Reply:
These metrics have for the library connect and for consumer/producer on the Kafka Connect. So it has _kafka.(.+)_ for takes the connect and consumer/producer. 

These metrics appear like this: 
Connect: 
```
kafka_connect_info{start_time_ms="1584551170124",client="connect-1",} 1.0
kafka_connect_info{commit_id="0bd886d86c8631d3",client="connect-1",} 1.0
kafka_connect_info{version="5.3.1-ccs",client="connect-1",} 1.0
```

Consumer: 
```
kafka_consumer_info{start_time_ms="1584551435732",client="consumer-5",} 1.0
kafka_consumer_info{commit_id="0bd886d86c8631d3",client="consumer-5",} 1.0
kafka_consumer_info{version="5.3.1-ccs",client="consumer-5",} 1.0
kafka_consumer_info{start_time_ms="1584551174695",client="consumer-2",} 1.0
kafka_consumer_info{commit_id="0bd886d86c8631d3",client="consumer-2",} 1.0
kafka_consumer_info{version="5.3.1-ccs",client="consumer-2",} 1.0
kafka_consumer_info{start_time_ms="1584551177275",client="consumer-3",} 1.0
kafka_consumer_info{commit_id="0bd886d86c8631d3",client="consumer-3",} 1.0
kafka_consumer_info{version="5.3.1-ccs",client="consumer-3",} 1.0
kafka_consumer_info{start_time_ms="1584551172188",client="consumer-1",} 1.0
kafka_consumer_info{commit_id="0bd886d86c8631d3",client="consumer-1",} 1.0
kafka_consumer_info{version="5.3.1-ccs",client="consumer-1",} 1.0
```

Producer: 
```
kafka_producer_info{start_time_ms="1584551435303",client="producer-6",} 1.0
kafka_producer_info{commit_id="0bd886d86c8631d3",client="producer-6",} 1.0
kafka_producer_info{version="5.3.1-ccs",client="producer-6",} 1.0
kafka_producer_info{start_time_ms="1584551440203",client="producer-7",} 1.0
kafka_producer_info{commit_id="0bd886d86c8631d3",client="producer-7",} 1.0
kafka_producer_info{version="5.3.1-ccs",client="producer-7",} 1.0
kafka_producer_info{start_time_ms="1584551176756",client="producer-3",} 1.0
kafka_producer_info{commit_id="0bd886d86c8631d3",client="producer-3",} 1.0
kafka_producer_info{version="5.3.1-ccs",client="producer-3",} 1.0
kafka_producer_info{start_time_ms="1584551438412",client="connector-producer-replicator-src-events-cluster-dest-aws-cluster-source-0",} 1.0
kafka_producer_info{commit_id="0bd886d86c8631d3",client="connector-producer-replicator-src-events-cluster-dest-aws-cluster-source-0",} 1.0
kafka_producer_info{version="5.3.1-ccs",client="connector-producer-replicator-src-events-cluster-dest-aws-cluster-source-0",} 1.0
kafka_producer_info{start_time_ms="1584551307711",client="producer-5",} 1.0
kafka_producer_info{commit_id="0bd886d86c8631d3",client="producer-5",} 1.0
kafka_producer_info{version="5.3.1-ccs",client="producer-5",} 1.0
kafka_producer_info{start_time_ms="1584551171716",client="producer-1",} 1.0
kafka_producer_info{commit_id="0bd886d86c8631d3",client="producer-1",} 1.0
kafka_producer_info{version="5.3.1-ccs",client="producer-1",} 1.0
kafka_producer_info{start_time_ms="1584551174227",client="producer-2",} 1.0
kafka_producer_info{commit_id="0bd886d86c8631d3",client="producer-2",} 1.0
kafka_producer_info{version="5.3.1-ccs",client="producer-2",} 1.0
```

Reply:
Start time should be a gauge - though the exporter will give them to you for free.

The label name here should be going in the metric name here, as the jmx exporter can't combine all of these into one info metric.